composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "phpol/string",
+    "description": "PHP Object Library String Component",
+    "type": "library",
+    "license": "MIT",
+    "minimum-stability": "stable",
+    "require": {},
+    "autoload": {
+        "psr-4": {
+            "OL\\String\\": "src/"
+        }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^6.0",
+        "friendsofphp/php-cs-fixer": "^2.0"
```

Comment:
Likewise, cs-fixer should be something we all have installed in our dev environments (something like phpcbf), not a part of the project, IMO.